clients/spark/core/src/main/hadoop3/scala/io/treeverse/clients/conditional/GCBackupAndRestore.scala

```diff
@@ -138,9 +184,9 @@ object GCBackupAndRestore {
 
   /** Required arguments are the following:
    *  1. address of parquet that includes the relative paths of files to backup or restore, created by a gc run
-   *  2. src: the namespace to backup or restore objects from/to (i.e. repo storage namespace, or an external location compatibly)
-   *  3. backup/restore destination: the namespace to backup or restore objects from/to (i.e. an external location or repo storage namespace compatibly)
-   *  4. Object storage type: "s3" or "azure"
+   *     2. src: the namespace to backup or restore objects from/to (i.e. repo storage namespace, or an external location compatibly)
+   *     3. backup/restore destination: the namespace to backup or restore objects from/to (i.e. an external location or repo storage namespace compatibly)
+   *     4. Object storage type: "s3" or "azure"
```

Comment:
I think this still clobbers indentation compared to "1" above.

Reply:
fixed